ChangeLog

```diff
@@ -13,6 +13,11 @@ Bugfix
 API Changes
    * Add a new function mbedtls_asn1_write_named_bitstring() to write ASN.1
      named bitstring in DER as required by RFC 5280 Appendix B.
+   * Deprecate the function mbedtls_x509write_csr_set_key_usage() and add a new
+     function mbedtls_x509write_csr_set_key_usage_ext() that has an unsigned
+     int input parameter instead of unsigned char to allow using the full 9
```

Comment:
Minor: `int` -> `integer`.

Reply:
No problem. I will add the flag that got lost to the ChangeLog. Do you think that fixing that is sufficient to get the PR approved?

Reply:
I just went over it again and added some nitpicks, but apart from that, the PR is ready for approval from my perspective.